go.mod

```diff
@@ -24,36 +23,56 @@ require (
 	github.com/tinkerbell/tink v0.0.0-20201207184750-9c52cca562bc
 	go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc v0.26.1
 	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.28.0
-	google.golang.org/grpc v1.41.0
+	google.golang.org/grpc v1.42.0
+)
+
+require (
```

Comment:
Why is the k8s stuff in a separate require block? Is that autogenerated?

Reply:
I honestly don't know. I've never seen `go` do this before. The indirect deps appearing separate was an addition in 1.17 I believe but this is peculiar.

Reply:
It might have been merge conflict stuff actually.